Makefile

```diff
@@ -8,7 +8,7 @@ test:
 	py.test $(target)
 
 coveragetest: clean
-	py.test --cov=$(target) $(target)
+	py.test --cov=$(target) --cov-report term --cov-report html $(target)
```

Comment:
"term" is the default, which I always follow with "coverage html" to generate the HTML report, which include annotations of the missed lines and branches. It is unneeded in TravisCI, though.